extension/lib/magento/customer/addItems.js

```diff
@@ -11,5 +11,11 @@ module.exports = async (context, input) => {
   const wishlistAddItemEndpointUrl = `${context.config.magentoUrl}/wishlists/${input.wishlistId}/items`
   const wishlistItems = input.transformedProducts.map(({ product }) => product)
 
-  await request.send(wishlistAddItemEndpointUrl, 'addFavorites', 'POST', wishlistItems)
+  const response = await request.send(wishlistAddItemEndpointUrl, 'addFavorites', 'POST', wishlistItems)
+  const wishlistItemIdMapping = await context.storage.user.get('wishlistItemIdMapping') || {}
```

Comment:
I believe this can be done a bit differntly, we have a storage map functionality that does this, the docs from shopify signify the usage, e.g. `storage.device.map...`:
```
@property {Object} map
 * @property {Function} map.get - (string mapName)
 * @property {Function} map.set - (string mapName, Object map)
 * @property {Function} map.del - (string mapName)
 * @property {Function} map.getItem - (string mapName, string key)
 * @property {Function} map.setItem - (string mapName, string key, string value)
 * @property {Function} map.delItem - (string mapName, string key)
```